xarray/core/variable.py

```diff
@@ -2570,6 +2575,11 @@ def chunk(  # type: ignore[override]
         dask.array.from_array
         """
 
+        if is_extension_array_dtype(self):
```

Comment:
```suggestion
        if is_extension_array_dtype(self._data.dtype):
```
We could also delete this since it isn't being tested (or is it)?